lib/elixir/lib/kernel.ex

```diff
@@ -2805,9 +2805,10 @@ defmodule Kernel do
   ## Module names
 
   A module name can be any atom, but Elixir provides a special syntax which is
-  usually used for module names. What is called a module name is an upper case
-  character followed by any alphanumeric characters or `_` or `.`. This
-  identifier is equivilant to an atom prefixed by `Elixir.`. So in the
+  usually used for module names. What is called a module name is an _upper case
+  Latin letter_ followed by any number of _Latin letters_, _numbers_, _underscores_
```

Comment:
Another term could be
https://en.wikipedia.org/wiki/Basic_Latin_(Unicode_block)#Uppercase_Latin_alphabet

* unaccented Latin letter
* Basic Latin Letter

the term ASCII is not used at all throughout elixir (there is only one mention in a comment in a test), and since we use Unicode, we should avoid it I think.

I think "unaccented Latin letter", "unaccented lowercase Latin letter", ""unaccented uppercase Latin letter"  sound natural to me. What do you think @jisaacstone